object/slicer/slicer.go

```diff
@@ -581,6 +581,10 @@ func (x *PayloadWriter) _writeChild(ctx context.Context, meta dynamicObjectMetad
 		payload := obj.Payload()
 		payloadAsBuffers := [][]byte{obj.Payload()}
 
+		if uint64(len(payload)) > x.payloadSizeLimit {
+			return fmt.Errorf("link's payload exceeds max available size: %d > %d", uint64(len(payload)), x.payloadSizeLimit)
```

Comment:
at this point we have complete chain accepted except linker, right? So, this makes the original object alive in the system, and we've done a lot of work. But here we fail. From one side, this looks pretty legit - we cannot form the complete chain. From the other one, we've made "enough" chain. What im thinking bout are:
1. will system auto-remove the chain eventually? this could be a huge pile of data user doesnt expect to reside in his container
2. if not, may be we'll let such chains to live and respond to user about this?

the case could be generalized to any problems with linker storage, but slicing is even more interesting

Reply:
We still have a valid object. Maybe it's better be controlled by some maximum object size limit (which can fail earlier than this one, but it'll leave incomplete chain then).

Reply:
there will always be (potential) situations when a user did not know the exact size and we have to stop splitting when some objects are already PUT, so i see no good solution except accepting them and marking garbage only if an additional user command is received (e.g. vacuum) OR sending some signed request from the user/slicer that the whole split was a mistake and parts should be marked for deletion